planemo_ext/galaxy/tools/linters/xml_order.py

```diff
@@ -17,16 +17,37 @@
     'citations',
 ]
 
+DATASOURCE_TAG_ORDER = [
+    'description',
+    'command',
+    'inputs',
+    'request_param_translation',
+    'uihints',
```

Comment:
Are uihints even respected for data source tools? Hmm... we can revisit that later though.

Can you add macros and configfiles back in the same order as normal tools?